README.md

```diff
@@ -63,60 +63,77 @@ Optimization simply use Ax.
 - scipy
 - multiple-dispatch
 
+### Prerequisite only for MacOS users:
+Before installing BoTorch, we recommend first manually installing PyTorch, a required dependency of BoTorch. Installing it according to the [PyTorch installation instructions](https://pytorch.org/get-started/locally/) ensures that it is properly linked against MKL, a library that optimizes mathematical computation for Intel processors. This will result in up to an order-of-magnitude speed-up for Bayesian optimization, as at the moment, installing PyTorch from pip does not link against MKL.
```

Comment:
We've generally tried to limit line lengths to around 100ish - mind doing this as well and adding a few breaks (here and below)? It's a lot easier to read in basic text editors/viewers that don't auto-wrap lines.